src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -410,10 +456,13 @@ private boolean addPlayer(@NotNull GameProfile gameprofile, Rank rank)
      * @param rank   Rank desired starting rank
      * @return True if successful, otherwise false
      */
-    public boolean addPlayer(@NotNull String player, Rank rank)
+    public boolean addPlayer(@NotNull String player, Rank rank, World world)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): world [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)